vpr/src/pack/constraints_report.cpp

```diff
@@ -0,0 +1,78 @@
+/*
+ * constraints_report.cpp
+ *
+ *  Created on: Dec. 7, 2021
+ *      Author: khalid88
+ */
+
+#include "constraints_report.h"
+
+struct region_content_info {
```

Comment:
comment this structure - vector that is accessed by the block type index, and a region

Reply:
no longer use this structure